runtime/src/bank.rs

```diff
@@ -1611,7 +1611,7 @@ impl Bank {
     pub fn verify_hash_internal_state(&self) -> bool {
         self.rc
             .accounts
-            .verify_hash_internal_state(self.slot(), &self.ancestors)
+            .verify_bank_hash(self.slot(), &self.ancestors)
```

Comment:
At this place, we semantically translate verify-ing Bank's `internal_state` into AccountDB's `bank_hash`. So rename propagation from AccountsDB ends here. :)